### PR TITLE
chore: Added crypto project

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Related Issue

Clears issue #1789 

## Description
Added a crypto currency project

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -4231,5 +4231,22 @@
...
+      "LinkedIn": "https://www.linkedin.com/in/karthik-nadar-b2155a25b/",
+      "Instagram": "https://www.instagram.com/karthiknadar1204/",
+      "Twitter": "https://twitter.com/Karthikreincar1"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/karthiknadar1204/Crypto",
+        "title": "CryptoWatchr",
+        "description": "CryptoWatchr is an innovative crypto project that provides users with a comprehensive platform to access and explore various cryptocurrency exchanges from around the world. Leveraging the power of the CoinGecko API users can seamlessly connect to different exchanges access real-time data and make informed decisions about their crypto investments.",
+        "tech": ["React js", "Chakra UI"]
```
Comment (priyankarpal) [inline DB/projects.json:4248]: change the tech stacks

```suggestion
        "tech": ["reactjs", "chakra ui"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by karthiknadar1204** [comment]
will do and push again

**Comment by karthiknadar1204** [comment]
Done @priyankarpal

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by karthiknadar1204** [comment]
@priyankarpal Please add the label

**Comment by priyankarpal** [comment]
> @priyankarpal Please add the label

We are not providing any labels for project additions as they are not associated with solving any issues.( according to gssoc23 rules)

**Pull request merged**